digdag-server/src/main/java/io/digdag/server/rs/RestModels.java

```
@@ -153,6 +239,32 @@ public static RestSession session(StoredSessionWithLastAttempt session, String p
                 .build();
     }
 
+    static RestSessionCollection sessionCollection(
+            ProjectStore ps, List<StoredSessionWithLastAttempt> sessions)
+    {
+        ProjectMap projs = ps.getProjectsByIdList(
+                sessions.stream()
+                        .map(Session::getProjectId)
+                        .collect(Collectors.toList()));
+
+        List<RestSession> collection = sessions.stream()
+                .map(session -> {
+                    try {
+                        return session(session, projs.get(session.getProjectId()).getName());
+                    }
+                    catch (ResourceNotFoundException ex) {
+                        // must not happen
+                        return null;
```

throw here instead of ignoring exception